client/src/interface/User.ts

```diff
@@ -1,6 +1,10 @@
 export interface User {
-  email: string;
   username: string;
+  email: string;
+  strategy: string;
+  id: number;
+  picture: string;
+  mongoId: number;
```

Comment:
You'll want to update this in the future to be a string because MongoIDs are strings. Also, where do we have two ids here?

Reply:
Id represents the value associated with the auth service (google and apple for the future). I kept them both in case we needed the mongo value to quickly access the data in database though I can remove that.